roles/kube-burner/templates/kubelet-density-heavy.yml.j2

```diff
@@ -0,0 +1,40 @@
+---
+global:
+  writeToFile: false
+{% if prometheus is defined and prometheus.prom_url is defined %}
+  indexerConfig:
+    enabled: true
+    esServers: [{{ prometheus.server }}]
+    insecureSkipVerify: true
+    defaultIndex: {{ workload_args.default_index|default("ripsaw-kube-burner") }}
+    type: elastic
+{% endif %}
+
+jobs:
+  - name: kubelet-density-heavy
```

Comment:
Can we add a way to label the nodes using custom nodeselector and max them out ( 250 pods per node ) taking into account the running pods on them like how we typically run kubelet density tests or is it already supported by tweaking the config?

Reply:
I think the user should just provide the label. I don't think Ripsaw needs to be applying labels to the nodes... It would need to understand "what" nodes the user wants... Meaning if their workers were not uniform, how would Ripsaw choose the right nodes to label? The user should simply provide the label to use, versus Ripsaw needing to encapsulate the logic of what nodes to label. Thoughts?

Reply:
Makes sense, we need to document it for the kubelet-density tests to make sure user adds a label to max out the pod capacity on selected nodes.

Reply:
The idea is to do this kind of actions from an e2e script. I haven't started coding it yet though

Reply:
Right but we also need to make sure it's documented in Ripsaw as well since we are explicitly supporting kubelet density workload IMHO.